### DISCUSSION TITLE
Digital Yacht AISnet working fine with latest AIS-catcher

### DISCUSSION THREAD

**Discussion by bertonumber1** [jvde-github/AIS-catcher#347] [General]
I came across an AISnet receiver on fleabay and it was faulty. i managed to get it working and despite its network module which does not seem to have power the USB serial is showing AIS messages and plotting them on catcher map via -e 38400 /dev/ttyACM0
line in command file. I am loving this new look with the choice of json or cmd configs. Just to let you know thats another one off the list as working "Digital Yacht".

Cheers again and all the best happy xmas and new year

Rab adsbgreenock

**Comment by jvde-github** [comment]
Thanks, much apreciated
